notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1345 @@
+{
```

Comment:
<p>Can we make the reference to <code>&nbsp;(see "Product quantization for nearest neighbor search" by Herve Jegou, Matthijs Douze, Cordelia Schmid). </code> into a link please? </p><p> </p><blockquote>), maps the vector y to the nearest cluster center. The number of&nbsp;clusters is n_lists. </blockquote><p>Please make the <code>n_lists</code> into an inline code sample for readability.  </p><p> </p><blockquote>  - a. "subspace" (C++ api: <code>codebook_gen::PER_SUBSPACE</code>): &nbsp;    creates <code>pq_dim</code> second-level quantizers - one for each slice of the data along features; &nbsp;  - b. "cluster" (C++ api: <code>codebook_gen::PER_CLUSTER</code>): &nbsp;    creates <code>n_lists</code> second-level quantizers - one for each first-level cluster. </blockquote><p>This isn't rendering as an enumerated list. Can you please explicitly add a line break?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>